backend/recipe/GroceryList.py

```diff
@@ -93,4 +93,4 @@ def purchased_item(id, item):
         #precondition: id as int and item as string
         #postcondition: removes item from grocery list, adds item to stock
         GroceryList.delete_item(id, item)
-        Stock.add_item(id, item)
\ No newline at end of file
+        Stock.Stock.add_item(id, item)
```

Comment:
found we can fix this by importing "from recipe.Stock import Stock" - This implementation works too, just looks a tad buggard